packs/sensu/etc/st2_handler.py

```diff
@@ -140,20 +142,25 @@ def _post_event_to_st2(url, body):
 
 
 def main(args):
-
     body = {}
     body['trigger'] = ST2_TRIGGERTYPE_REF
     body['payload'] = json.loads(sys.stdin.read().strip())
     _post_event_to_st2(_get_st2_webhooks_url(), body)
 
-
 if __name__ == '__main__':
+    if len(sys.argv) > 1:
```

Comment:
argparse would be nice with positional args. Its ok if you are not motivated I can add that in later. (backlog)